src/components/Popup/Popup.scss

```diff
@@ -1,14 +1,21 @@
 .tm-popup {
   div:first-child {
```

Comment:
please let's not style div's without class names or ids.

Reply:
yes, the popup css is not ideal in react-spatial that's why there is this div:first-child.